internal/ingress/controller/store/backend_ssl.go

```diff
@@ -148,7 +154,7 @@ func (s k8sStore) checkSSLChainIssues() {
 			continue
 		}
 
-		data, err := ssl.FullChainCert(secret.PemFileName, s.filesystem)
+		data, err := ssl.FullChainCert([]byte(secret.PemCertKey))
```

Comment:
@hnrytrn 👋 Could you explain why this change was necessary? I think I'm missing a beat.

Reply:
One of the changes I added is that the PEM certificate isn't written on disk during dynamic configuration mode, so `PemFileName` would be empty.

Reply:
Ah of course :)